tests/test_pages.py

```diff
@@ -12,6 +12,11 @@
 from selenium.common.exceptions import TimeoutException
 
 
+EXCLUDE_ERRORS = (
+    'Scrollbar test exception: TypeError: Cannot read properties of',
```

Comment:
Is there a known issue for this? I'd like to add at least add a comment tracking it.